lib/opensslca.pm

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Base module for OpenSSL Certificate Authority
+#          The corresponding key pairs can be use in
+#          a number of situations, such as issuing server
+#          certificates to secure an intranet website, or
+#          for issuing certificates to clients to allow them
+#          to authenticate to a server
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#88513, tc#1768672
+
+package opensslca;
+
+use base Exporter;
+use Exporter;
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+
```

Comment:
One space line is good, delete the surplus ones

Reply:
Let me fix it

Reply:
Thanks Lili, fixed.